pkg/haproxy/types/types.go

```diff
@@ -89,6 +89,7 @@ type Global struct {
 	CustomHTTPHAResponses   []HTTPResponse
 	CustomSections          []string
 	CustomTCP               []string
+	UseCoraza               bool
```

Comment:
Better moving to ModSecurity struct.

https://github.com/jcmoraisjr/haproxy-ingress/blob/635669c5debb1b479117384c62f6171fff4dc3e6/pkg/haproxy/types/types.go#L182-L186